atlasreader/atlasreader.py

```diff
@@ -673,21 +670,35 @@ def create_output(filename, atlas='all', voxel_thresh=1.96, cluster_extent=20,
                                   cmap=plotting.cm.cold_hot,
                                   output_file=glass_fname)
 
-    # get template image for plotting cluster maps
-    bgimg = nb.load(
-        resource_filename(
-            'atlasreader',
-            'data/templates/MNI152_T1_1mm_brain.nii.gz'
+    # Check if thresholded image contains only zeros
+    if np.count_nonzero(thresh_img.get_data()) == 1:
```

Comment:
If I'm reading this correctly, this will only trigger if `thresh_img` has a single value... I think we might just want `if np.any(thresh_img.get_data()):` since we don't actually care about the number of zeros, but rather that there is at least one value that _isn't_ zero.

Reply:
uiuiui... I shouldn't look at this code anymore. yes, this only works for a single value.
About `np.any`, doesn't that give you an array back with indices?
Interesting: `array([])) == False` is correct.

Reply:
`np.any` returns a boolean value, so I think it would work well in this `if` clause.